app/main.py

```diff
@@ -2,24 +2,69 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int,
+                 max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner, name) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance, owner):
+        getattr(instance, self.protected_name)
```

Comment:
The `__get__` method should return the value of the attribute. Currently, it only calls `getattr` without returning anything. You should return the result of `getattr(instance, self.protected_name)`.